src/Document/Library/Bridge/Symfony/DependencyInjection/ZenstruckDocumentLibraryExtension.php

```diff
@@ -90,6 +92,19 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
             ->addTag('form.type')
         ;
 
+        // value resolver
+        $container->register(RequestFilesExtractor::class)
```

Comment:
```suggestion
        $container->register('.zenstruck_document.value_resolver.request_files_extractor', RequestFilesExtractor::class)
```

(otherwise this service will be autoloadable.